config/crd/bases/postgres-operator.crunchydata.com_postgresclusters.yaml

```diff
@@ -4688,6 +4689,52 @@ spec:
                           type: object
                       type: object
                     type: array
+                  parameters:
+                    additionalProperties:
+                      anyOf:
+                      - type: integer
+                      - type: string
+                      x-kubernetes-int-or-string: true
+                    description: |-
+                      Configuration parameters for the PostgreSQL server. Some values will
+                      be reloaded without validation and some cause PostgreSQL to restart.
+                      Some values cannot be changed at all.
+                      More info: https://www.postgresql.org/docs/current/runtime-config.html
+                    maxProperties: 50
+                    type: object
```

Comment:
❓ We should be explicit: is this field atomic or granular?

Reply:
I'm thinking granular makes sense.